CatAnalyzer/test/run_TtbarDiLeptonAnalyzer_cfg.py

```diff
@@ -47,7 +47,7 @@
         jetConeSize = cms.double(0.4),
     )
 
-process.filterRECO = cms.EDProducer("CATTriggerBitCombiner",
+process.filterRECO = cms.EDFilter("CATTriggerBitCombiner",
     triggerResults = cms.InputTag("TriggerResults::CAT"),
```

Comment:
can u change this to TriggerResults::PAT and have secondaryTriggerResults as TriggerResults::RECO